includes/classes/Indexable/Post/Post.php

```diff
@@ -385,9 +385,25 @@ public function get_indexable_post_taxonomies( $post ) {
 		 * @param  {WP_Post} Post object
 		 * @return  {array} New taxonomies
 		 */
-		$selected_taxonomies = apply_filters( 'ep_sync_taxonomies', $selected_taxonomies, $post );
+		$selected_taxonomies = (array) apply_filters( 'ep_sync_taxonomies', $selected_taxonomies, $post );
 
-		return $selected_taxonomies;
+		// Important we validate here to ensure there are no imposters, as just one would cause wp_get_object_terms() to fail.
```

Comment:
```suggestion
		// Important we validate here to ensure there are no invalid taxonomy values returned from the filter, as just one would cause wp_get_object_terms() to fail.
```

Reply:
![amongus](https://user-images.githubusercontent.com/8536129/95897125-a1534380-0d52-11eb-9f39-847f3b693453.jpg)